tests/data/FactoryUtils.ts

```diff
@@ -8,8 +8,9 @@ export default class FactoryUtils {
     return values[i];
   }
 
-  public static getRandomNumber(range: number, minimum = 0, scale = 1): number {
-    return (Math.floor(Math.random() * range) % scale) + minimum;
```

Comment:
this function wasn't working for ranges with `scale = 1` (since any integer mod 1 is 0, so it would always return `minimum`), so I refactored it a bit

Reply:
nit: rename `range` to `value` or something